client/src/components/podcasts/PodcastShowPage.js

```diff
@@ -2,15 +2,15 @@ import React, { useState, useEffect } from "react"
 import ErrorList from "../ErrorList"
 import translateServerErrors from "../../services/translateServerErrors"
 
-import PodcastReview from './PodcastReviewForm'
+import PodcastReviewForm from './PodcastReviewForm'
 import ReviewTile from './ReviewTile.js'
 
 const PodcastShowPage = (props) => {
   const [podcast, setPodcast] = useState({
     name: "",
-    description: "",
-    reviews: []
+    description: ""
   })
+  const [reviews, setReviews] = useState([])
```

Comment:
If a Podcast has Reviews we can track them both in the same piece of state like you had before
```
useState({
  name: "",
  description: "",
  reviews: []
})
```